spot-setup/hdfs_setup.sh

```diff
@@ -7,8 +7,8 @@ source /etc/spot.conf
 #
 # creating HDFS user's folder
 #
-sudo -u hdfs hadoop fs -mkdir ${HUSER}
-sudo -u hdfs hadoop fs -chown ${USER}:supergroup ${HUSER}
+hadoop fs -mkdir ${HUSER}
```

Comment:
so what was the issue... that hadoop had access permissions but hdfs and duxbury didn't, and the old code got around that with a sudo but then you tried running it someplace where duxbury didn't have sudo privileges?